docs/application/native/guides/ui/tizen-ws-shell.md

```diff
@@ -0,0 +1,155 @@
+
+**Figure: Layer diagram for the Tizen window system shell**
+
+![Layer diagram for the Tizen window system shell](./media/tzsh_arch.png)
+
+To support each system GUI service in Tizen, the TZSH provides the following libraries:
+
+- **tzsh-quickpanel**: For the quickpanel window which shows notifications and system setup widgets.
+
+- **tzsh-softkey**: For the softkey window which shows the device back and home buttons.
+
+Not all applications require to use the TZSH's functionalities. However, in some cases, certain applications may require to perform manipulation of system GUI service's window. For example, media player application needs to close the quickpanel window during playback of video. In this case, you can use the tzsh-quickpanel library.
+
+
+## Prerequisites
+
+To use the functions and structures of the TZSH quickpanel API (in [mobile](../../api/mobile/latest/group__TIZEN__WS__SHELL__QUICKPANEL__MODULE.html) and [wearable](../../api/wearable/latest/group__TIZEN__WS__SHELL__QUICKPANEL__MODULE.html) applications), include the `<tzsh_quickpanel.h>` header file in your application:
+
+```
+#include <tzsh_quickpanel.h>
+```
+
+
+## Create TZSH Quickpanel Handler
+Once you have created the main window for your application, call the `tzsh_quickpanel_create_with_type` function with a native window ID to create `tzsh_quickpanel_h` structure (in [mobile](../../api/mobile/latest/group__TIZEN__WS__SHELL__QUICKPANEL__MODULE.html#gaaa00e8e25b43c9538ca188bc43bdb3ac) and [wearable](../../api/wearable/latest/group__TIZEN__WS__SHELL__QUICKPANEL__MODULE.html#gaaa00e8e25b43c9538ca188bc43bdb3ac) applications).
+
+```
+#include <Elementary.h>
+
+static void
+init(const char *name)
+{
+   Evas_Object *main_win;
+
+   /* create main window for the application */
+   main_win = elm_win_util_standard_add(name, name);
+
+   /* set up main window */
+   ...
+   evas_object_show(main_win);
+
+
+   tzsh_h tzsh;
+   tzsh_quickpanel_h tzsh_qp;
+   tzsh_window tz_win;
+
+   /* Get native window ID of main window */
+   tz_win = elm_win_window_id_get(main_win);
+
+   /* Create tzsh_h structure */
+   tzsh = tzsh_create(TZSH_TOOLKIT_TYPE_EFL);
+
+   /* Create tzsh_quickpanel_h structure */
+   tzsh_qp = tzsh_quickpanel_create_with_type(tzsh, tz_win, TZSH_QUICKPANEL_TYPE_SYSTEM_DEFAULT);
+```
+
+
+## Show or Hide Quickpanel Window
+
+To show or hide quickpanel window when your application's window is activated, call the following functions:
+
+```
+   if (show)
+     tzsh_quickpanel_show(tzsh_qp); /* To show quickpanel window */
+   else
+     tzsh_quickpanel_hide(tzsh_qp); /* To hide quickpanel window */
+```
+
+## Get Visiblility Status of Quickpanel Window
```

Comment:
Please check "Visiblility".
It looks misspell of Visibility.

Reply:
@gl-lee 

Get Visibility Status of Quickpanel Window

Reply:
done.